### DISCUSSION TITLE
The development branch ready for ceating feature branch.

### DISCUSSION THREAD

**Discussion by ktlau** [ktlau/Tafexxxxyy#1] [General]
The development branch has compltede with the main menu. Each individual member may start cloing the gitHub repository for creating feature branch based on it.

**Comment by kwokLau** [comment]
I have created the feature branch for the mortgage calculator based on the development. Have you started creating your feature branch yet. Make sure that it is based on the development branch.

**Comment by ktlau** [comment]
I have also created the foreign exchange calculator feature branch based on the development branch.

**Reply by kwokLau** [reply]
Thx. Let's develop the codes in the feature branches. I'll let you know when I have completed mine.

**Comment by ktlau** [comment]
I have completed the codng for the currency conversion calculator page. You may start the merging of the mortgage calculator page into the development branch.

**Reply by kwokLau** [reply]
OK. I have completed the coding of the mortgage calculator. I am goign to do the merging to the development branch first and will be you know when it is done.
